src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml

```diff
@@ -336,13 +336,12 @@
                     <Insets bottom="4.0" left="4.0" right="4.0" top="4.0"/>
                   </padding>
                 </HBox>
-                <HBox alignment="CENTER_LEFT" spacing="4.0">
-                  <Label text="Code" prefWidth="50.0"/>
-                  <TextField prefHeight="25.0" prefWidth="100.0" fx:id="textFieldCandidateCode"/>
+                <Label text="Aliases, codes, and other spellings (one per line):">
```

Comment:
Nit: remove colon, since we don't use it for other fields.

Reply:
This is the first textarea I believe. The other fields are labels on a field on the same line, so it's more clear without a semicolon.

I don't have extremely strong opinions on this, but I do want to note that this was intentional: I tried it without a semicolon at first, and this looked better to my eye. Happy to remove it, though -- let me know!

Reply:
I'm ok with leaving it given that justification. :)